.github/workflows/unittests.yml

```diff
@@ -49,7 +49,7 @@ jobs:
       - name: Install Gluten Arrow-backend
         run: |
           rm -rf ~/.m2/repository/org/apache/arrow && rm -rf ~/.m2/repository/io/glutenproject/
-          mvn clean package -Pspark-3.2 -Pbackends-gazelle -DskipTests -Dbuild_cpp=ON -Dbuild_gazelle_cpp=ON -Dbuild_arrow=ON
+          mvn clean package -Pspark-3.2 -Pbackends-gazelle -DskipTests -Dbuild_cpp=ON -Dbuild_gazelle_cpp_backend=ON -Dbuild_arrow=ON
```

Comment:
Let's change gazelle to arrow. Currently we didn't and won't use gazelle anymore.

Reply:
Ok, will fix it in another PR.